pkg/virt-controller/watch/vm_test.go

```diff
@@ -1011,3 +1045,15 @@ func DefaultVirtualMachineWithNames(started bool, vmName string, vmiName string)
 func DefaultVirtualMachine(started bool) (*v1.VirtualMachine, *v1.VirtualMachineInstance) {
 	return DefaultVirtualMachineWithNames(started, "testvmi", "testvmi")
 }
+
+func getVmCondition(vm *virtv1.VirtualMachine, condType virtv1.VirtualMachineConditionType) *virtv1.VirtualMachineCondition {
```

Comment:
why is it needed? can't we use `getCondition`? https://github.com/kubevirt/kubevirt/pull/3176/files#diff-3d28bf4913d346c1909faf1a6411f176R48

Reply:
Moved the function to a common place `pkg/controller/conditions.go` and it is called when needed.